### PR TITLE
Update remaining old icons

### PR DESCRIPTION
**Pull request by MariagraziaAlastra**
**Reviewer:** @laurengarcia 

## Description:
I just noticed while testing the extension, some of the old icons in the repo weren't updated (we didn't have new assets for them), so in the trackers subview they looked broken, because:
1. We see we have the icon available and we try to render it
2. It's the old icon with the grey background and transparent fill, so it just shows up as a solid grey circle, looking broken

Uploaded the new icons in this PR and updated the styles.
We now have a different set for the top offenders pill version, and for the tracker networks subview.
The first are the tn-*.svg icons, the subview ones are the tn-g-*.svg ones (g is for grey).

screen shot 2018-01-04 at 11 24 22 am

## Steps to test this PR:
1. Build and reload extension
2. Test websites such as huffingtonpost.com, 
4. Pill version is unchanged
5. Tracker networks subview has the grey background icons as expected, and salesforce, openx, criteo and casale show up normally

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/scss/_tracker-network-icons.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Tracker Network Icons */
-$icons: ("adobe", "adtech", "amazon", "aol", "appnexus", "casale", "chartbeat", "comscore", "criteo", "facebook", "fox", "generic", "google", "mixpanel", "moat", "newrelic", "nielsen", "openx", "outbrain", "quantcast", "salesforce", "taboola", "twitter", "yahoo", "yandex");
+$icons: ("adobe", "adtech", "amazon", "aol", "appnexus", "chartbeat", "comscore", "facebook", "fox", "generic", "google", "mixpanel", "moat", "nielsen", "outbrain", "quantcast", "taboola", "twitter", "yahoo", "yandex");
+$grey-icons: ("adobe", "adtech", "amazon", "aol", "appnexus", "casale", "chartbeat", "comscore", "criteo", "facebook", "fox", "google", "mixpanel", "moat", "newrelic", "nielsen", "openx", "outbrain", "quantcast", "salesforce", "taboola", "twitter", "yahoo", "yandex");
```
Comment (MariagraziaAlastra) [inline shared/scss/_tracker-network-icons.scss]: Different list since we have different sets for the pill version and for the subview.

Comment (laurengarcia) [reply shared/scss/_tracker-network-icons.scss]: Can you give these a variable that is descriptive of its function vs. appearance?  Perhaps `$subview-icons` or something like that.

Comment (laurengarcia) [reply shared/scss/_tracker-network-icons.scss]: Also, what does the `-g-` in the middle of the .svg file names indicate? i.e. `shared/img/tn-g-twitter.svg` Can this also be more descriptive? Nobody is going to know what -g- means.

Comment (MariagraziaAlastra) [reply shared/scss/_tracker-network-icons.scss]: g is for grey, but yeah you're right, maybe tn-subview-*.svg would work better

**Comment by MariagraziaAlastra** [comment]
@laurengarcia ready again, I renamed the icons and the variable 👍

**Pull request merged**